server/main.js

```diff
@@ -511,6 +513,15 @@ app.put('/share', upload.single('file'), async function(req, res) {
       res.status(400).json({success: false, message: "Missing file"})
       return
     }
+
+    if(file.mimetype == "image/jpeg") {
+        // Remove EXIF data
+        console.log("User is uploading a JPEG, removing EXIF data")
+        let imageOld = "data:image/jpeg;base64," + file.buffer.toString("base64")
+        let imageNew = piexif.remove(imageOld).substring(imageOld.indexOf(','))
```

Comment:
@MegaApuTurkUltra Ready to merge whenever, unless you'd like that `console.log` removed or changed.